src/main/java/io/confluent/connect/jdbc/source/TableMonitorThread.java

```diff
@@ -105,6 +124,10 @@ public void run() {
     if (tables == null) {
       throw new ConnectException("Tables could not be updated quickly enough.");
     }
+    if (!duplicates.isEmpty()) {
+      log.info("The filtered tables are {} ", tables);
```

Comment:
Should this go outside of the `if` block? Seems useful whether or not there are duplicates. Thoughts?

Reply:
Its already logged in updateTables in debug. Maybe we should change that to info. Seems, pretty useful info to me.

Reply:
I agree its useful to have INFO, but shouldn't it happen every time? IOW, I thought you were going to remove this log message and change the one in `updateTables` to INFO?

Reply:
I agree its useful to have INFO, but shouldn't it happen every time? IOW, I thought you were going to remove this log message and change the one in updateTables to INFO?